From eb85adc634093f59e9e5ed7ebb52394bed7ecf24 Mon Sep 17 00:00:00 2001
From: Robin Lee <cheeselee@fedoraproject.org>
Date: Wed, 31 Jan 2018 11:25:42 +0800
Subject: [PATCH] Change metainfo ID to 'org.zim_wiki.Zim'

Hyphens are discouraged to be used with respect to recent version of AppStream,
and 'org.zim-wiki.Zim' failed to be a valid Flatpak ID.
---
 setup.py                                                             | 2 +-
 xdg/{org.zim-wiki.Zim.metainfo.xml => org.zim_wiki.Zim.metainfo.xml} | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)
 rename xdg/{org.zim-wiki.Zim.metainfo.xml => org.zim_wiki.Zim.metainfo.xml} (98%)

diff --git a/setup.py b/setup.py
index 240aa526..776baa68 100755
--- a/setup.py
+++ b/setup.py
@@ -87,7 +87,7 @@ def collect_data_files():
 		('share/applications', ['xdg/zim.desktop']),
 		('share/mime/packages', ['xdg/zim.xml']),
 		('share/pixmaps', ['xdg/hicolor/48x48/apps/zim.png']),
-		('share/metainfo', ['xdg/org.zim-wiki.Zim.metainfo.xml']),
+		('share/metainfo', ['xdg/org.zim_wiki.Zim.metainfo.xml']),
 	]
 
 	# xdg/hicolor -> PREFIX/share/icons/hicolor
diff --git a/xdg/org.zim-wiki.Zim.metainfo.xml b/xdg/org.zim_wiki.Zim.metainfo.xml
similarity index 98%
rename from xdg/org.zim-wiki.Zim.metainfo.xml
rename to xdg/org.zim_wiki.Zim.metainfo.xml
index 6b98d3a4..c0097ac7 100644
--- a/xdg/org.zim-wiki.Zim.metainfo.xml
+++ b/xdg/org.zim_wiki.Zim.metainfo.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2014 Ryan Lerch <rlerch@redhat.com>, 2018 Jaap Karssenberg <jaap.karssenberg@gmail.com> -->
 <component type="desktop">
-  <id>org.zim-wiki.Zim</id>
+  <id>org.zim_wiki.Zim</id>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
 	<name>Zim</name>
-- 
2.13.6

